crates/typst-library/src/layout/grid/mod.rs

```diff
@@ -36,7 +36,11 @@ use crate::visualize::{Paint, Stroke};
 /// in the document flow. Set and show rules on one of these elements do not
 /// affect the other.
 ///
+/// # Sizing the tracks { #track-size }
+///
 /// A grid's sizing is determined by the track sizes specified in the arguments.
+/// There are many such sizing parameters: [`columns`]($grid.columns),
+/// [`rows`]($grid.rows), [`gutter`]($grid.gutter), etc. and those of [table].
```

Comment:
Table is irrelevant here, it is already mentioned above that table works the same way in terms of layout, but for a different purpose.

```suggestion
/// There are multiple sizing parameters: [`columns`]($grid.columns),
/// [`rows`]($grid.rows) and [`gutter`]($grid.gutter).
```